Automattic/jetpack
# Jetpack Sync: Add/sync woo remove order items

Add the `woocommerce_remove_order_items` event to Jetpack Sync, added on WooCommerce on https://github.com/woocommerce/woocommerce/pull/37822 . This PR also needs changes on D126373-code to whitelist the event on WPCOM (already merged)

## Proposed changes:
* Add action listener `woocommerce_remove_order_items`
* Get the order items id and send them as a group using action `woocommerce_remove_order_item_ids`

### Other information:


## Does this pull request change what data or activity we track or use?



## Testing instructions:
Test plan for Automatticians
* Create a test site with Payments configured. Probably JN is the simpler option (you will need to enable global-http)
** Include WooCommerce
** Include WooCommerce Payments
** Include WooCommerce Payments Dev Tools
** Include WooCommerce Payments JN Options
** Jetpack Beta with this branch (add/sync-woo-remove-order-items)
** Add a Backup plan
* ~~Apply D126373-code and configure your site to use your sandbox.~~ (already merged)
* As an admin, add some products to your website
* As a customer, add the products to your shopping cart and go to the checkout 
* Use a declined credit card to pay https://woocommerce.com/document/woopayments/testing-and-troubleshooting/testing/#declined-test-cards
* Verify you received the events `woocommerce_remove_order_item` with the order_item_id. You could do it by adding a log with the events before they are sent to backups (fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Swrgcnpx%2Sflap%2Spynff.wrgcnpx%2Qflap%2Qerjvaq.cuc%3Se%3Q29612p98%23230-og)
```
error_log( '-->send_events_to_backup:' . json_encode( $this->events ) );
```
